R/getAge.R

```diff
@@ -1,131 +1,127 @@
 #' Get age information for each fish
 #'
-#' Get age information for each fish, or row of the data.
-#' This functionality was originally in cleanAges, moved to
-#' cleanPacFIN, and is now compartmentalized here, but called in
-#' [cleanPacFIN] and stored in the `Age` column.
+#' Get age information for each fish, or row of the data. This functionality was
+#' originally in cleanAges, moved to cleanPacFIN, and is now compartmentalized
+#' here, but called in [cleanPacFIN] and stored in the `Age` column.
 #'
 #' @details
-#' Age types listed in keep (or derivatives of those) are used to filter
-#' for ages that should be kept, i.e., not returned as `NA`.
-#' If `FISH_AGE_YEARS_FINAL` doesn't have an entry, where this should be the
-#' best age available, then the age reads in `age1`, `age2`, ... are
-#' averaged over and rounded to the next integer
-#' (i.e., [ceiling]) if their methods are acceptable.
-#' When `FISH_AGE_YEARS_FINAL` does have an entry, this age is kept if one of
-#' the methods in `AGE_METHOD` is listed in `keep`. It is currently unclear
-#' how `FISH_AGE_YEARS_FINAL` is determined because it doesn't always equal
-#' the value from the first age reader. Sometimes, it is `ceiling(mean())`,
-#' other times it appears to default to break and burn, and other random combos.
-#' We use the best judgment of those who created the column that they know what
-#' they are doing, but please feel free to report errors in PacFIN to the
-#' github issues or the package maintainer after contacting the state representative.
+#' Age types listed in keep (or derivatives of those) are used to filter for
+#' ages that should be kept, i.e., not returned as `NA`. If
+#' `FISH_AGE_YEARS_FINAL` doesn't have an entry, where this should be the best
+#' age available, then the user is warned that they should potentially look at
```

Comment:
@kellijohnson-NOAA The warning is no longer produced. It used to be some text

"Investigate the following SAMPLES for errors because
FINAL_FISH_AGE_IN_YEARS doesn't match the mean of all reads:" 

and then a whole long series of output. I did not like the long output so glad that is gone. To keep this comment correct, I suggest outputting a line to at minimum warn the user there are samples without a final age: something like "XX samples have no FISH_AGE_YEARS_FINAL and are not assigned an 'Age'", that or remove this comment that a warning is produced.

Reply:
Thanks @brianlangseth-NOAA I just changed this to add another bullet point to the output providing a count.